packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -383,25 +398,27 @@
             }
         }
         &__text {
+            display: inline-block;
             padding: 1.6rem 1.6rem 0;
-            font-size: var(--text-size-xxs);
-            color: var(--text-disabled);
-            line-height: var(--text-lh-l);
-            font-weight: var(--text-weight-normal);
-            text-align: var(--text-align-left);
+            @include mobile {
+                padding: 1.6rem 0 0.8rem 1.6rem;
+                font-size: var(--text-size-s);
```

Comment:
@kate-deriv as i understand, the size of text will change for mobile, right? so in this case we can use <Text size={isMobile() ? 's' : 'xs'} for this purpose instead of changing the fonts in scss👍🏻 applicable to lines 441-442 too )

Reply:
Oh, please, excuse me, I didn't noticed(